index.html

```diff
@@ -289,6 +289,36 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <section class="contact_form">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in <span>touch.</span>
+        </p>
+        <form action="#" class="form" method="POST">
```

Comment:
- [ ] Good Job Implementing the Form, but we missed a few details 🤏 form action attribute should be similar to https://formspree.io/f/{form_id} and the method attribute should equal `POST`. According to the requirements you should use [FormSpree](https://formspree.io/html/)

![data](https://user-images.githubusercontent.com/45231276/175351955-71713844-4fea-4aec-9865-bc76e4ca4cdd.PNG)